openstf/model/objective.py

```diff
@@ -0,0 +1,105 @@
+# SPDX-FileCopyrightText: 2017-2021 Alliander N.V. <korte.termijn.prognoses@alliander.com> # noqa E501>
+#
+# SPDX-License-Identifier: MPL-2.0
+import optuna
+
+from openstf.enums import MLModelType
+from openstf.model.model_creator import ModelCreator
+from openstf.model_selection.model_selection import split_data_train_validation_test
+from openstf.metrics.metrics import mae
+
+EARLY_STOPPING_ROUNDS: int = 10
+TEST_FRACTION: float = 0.1
+VALIDATION_FRACTION: float = 0.1
+# See https://xgboost.readthedocs.io/en/latest/parameter.html for all possibilities
+EVAL_METRIC: str = "mae"
+
+# https://optuna.readthedocs.io/en/stable/faq.html#objective-func-additional-args
+class XGBRegressorObjective:
```

Comment:
Nice to have these links in the documentation

Reply:
thx. I like to add links if its not obvious and or if I think I need to look it up later perhaps.